acquia/acquia_cms
# ACMS-1108: Cleanup acquia/acquia_cms module.

**Motivation**
Fixes #[ACMS-1108](https://backlog.acquia.com/browse/ACMS-1108)

**Proposed changes**
Cleanup acquia_cms module

**Alternatives considered**
Please merge [this](https://github.com/acquia/acquia_cms/pull/1150) PR first in 2.0.x branch

**Testing steps**
- install site with minimal profile
- require acms module with this branch
- enable acms module
- verify site working well 

**Merge requirements**
